spring-cloud-gcp-data-spanner/src/test/java/org/springframework/cloud/gcp/data/spanner/core/convert/ConverterAwareMappingSpannerEntityReaderTest.java

```diff
@@ -70,12 +75,37 @@ public void readNestedStructTest() {
 						ImmutableList.of(innerStruct))
 				.build();
 
-		OuterTestEntity result = this.readConverter.read(OuterTestEntity.class, outerStruct);
+		OuterTestEntity result = this.spannerEntityReader.read(OuterTestEntity.class,
+				outerStruct);
 		assertEquals("key1", result.id);
 		assertEquals(1, result.innerTestEntities.size());
 		assertEquals("value", result.innerTestEntities.get(0).value);
 	}
 
+	@Test
+	public void readConvertedNestedStructTest() {
+		Struct innerStruct = Struct.newBuilder().add("value", Value.string("value"))
```

Comment:
nit suggestions: 
* rename "value" -> "string_col" 
* rename innerStruct to structColumn
* rename outerStruct to rowStruct

Reply:
done